### PR TITLE
updated badges

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [ ] Link to relevant GitHub issues or pull requests

- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

This way we don't have to manage the docs after each release :sweat_smile:

### CODE REVIEW

File: docs/PLUGINS.md [hunk 1]
```diff
@@ -198,9 +198,11 @@ Simplest option for you to test JCasC compatibility in your plugin is to introdu
 
 Add the Configuration as Code plugin as a test dependency in your pom.xml:
 
+[LATEST_RELEASE](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
```
Comment (timja) [inline docs/PLUGINS.md]: might be a badge url you can use for this?

Comment (casz) [reply docs/PLUGINS.md]: Will this do? :sweat_smile: 
latest release

Comment (timja) [reply docs/PLUGINS.md]: What about:
```
[![latest release](https://img.shields.io/github/release/jenkinsci/configuration-as-code-plugin.svg)](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
```

Comment (timja) [reply docs/PLUGINS.md]: Same badge but linking to the release

Comment (casz) [reply docs/PLUGINS.md]: I'll guess I have to badge the readme too :laughing:

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+[![Contributors](https://img.shields.io/github/contributors/jenkinsci/configuration-as-code-plugin.svg)](https://github.com/jenkinsci/configuration-as-code-plugin/graphs/contributors)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/1c872818b46f4fdd890e4a22af0bee8c)](https://www.codacy.com/app/casz/configuration-as-code-plugin)
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/configuration-as-code.svg)](https://plugins.jenkins.io/configuration-as-code)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/configuration-as-code-plugin.svg?label=release)](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/i/configuration-as-code.svg?color=blue)](https://plugins.jenkins.io/configuration-as-code)
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
+[![Jenkins Version](https://img.shields.io/badge/Jenkins-2.60.3-yellow.svg)](https://jenkins.io/download/)
+![JDK8](https://img.shields.io/badge/jdk-8-yellow.svg)
```
Comment (timja) [inline README.md]: drop this?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+[![Contributors](https://img.shields.io/github/contributors/jenkinsci/configuration-as-code-plugin.svg)](https://github.com/jenkinsci/configuration-as-code-plugin/graphs/contributors)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/1c872818b46f4fdd890e4a22af0bee8c)](https://www.codacy.com/app/casz/configuration-as-code-plugin)
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/configuration-as-code.svg)](https://plugins.jenkins.io/configuration-as-code)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/configuration-as-code-plugin.svg?label=release)](https://github.com/jenkinsci/configuration-as-code-plugin/releases/latest)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/i/configuration-as-code.svg?color=blue)](https://plugins.jenkins.io/configuration-as-code)
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
+[![Jenkins Version](https://img.shields.io/badge/Jenkins-2.60.3-yellow.svg)](https://jenkins.io/download/)
```
Comment (timja) [inline README.md]: drop this? not that useful

File: README.md [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
+![JDK8](https://img.shields.io/badge/jdk-8-yellow.svg)
+[![License: MIT](https://img.shields.io/badge/license-MIT-yellow.svg)](LICENSE)
```
Comment (timja) [inline README.md]: drop this? github does a good job of showing it

File: README.md (hunk 2 above)
Comment (casz) [reply README.md]: I throw everything in to see what would stick :trollface:

**Pull request merged**